hpack/src/main/java/com/twitter/hpack/StaticTable.java

```diff
@@ -29,22 +29,22 @@
   // Appendix A: Static Table
   // http://tools.ietf.org/html/rfc7541#appendix-A
   private static final List<HeaderField> STATIC_TABLE = Arrays.asList(
-    /*  1 */ new HeaderField(":authority", EMPTY),
-    /*  2 */ new HeaderField(":method", "GET"),
-    /*  3 */ new HeaderField(":method", "POST"),
-    /*  4 */ new HeaderField(":path", "/"),
-    /*  5 */ new HeaderField(":path", "/index.html"),
-    /*  6 */ new HeaderField(":scheme", "http"),
-    /*  7 */ new HeaderField(":scheme", "https"),
-    /*  8 */ new HeaderField(":status", "200"),
-    /*  9 */ new HeaderField(":status", "204"),
-    /* 10 */ new HeaderField(":status", "206"),
-    /* 11 */ new HeaderField(":status", "304"),
-    /* 12 */ new HeaderField(":status", "400"),
-    /* 13 */ new HeaderField(":status", "404"),
-    /* 14 */ new HeaderField(":status", "500"),
-    /* 15 */ new HeaderField("accept-charset", EMPTY),
-    /* 16 */ new HeaderField("accept-encoding", "gzip, deflate"),
+    /*  1 */ new HeaderField(":authority", EMPTY, ":authority"),
```

Comment:
Will tidy these up. Any objection to using factory methods on HeaderField to make the construction use-cases clearer?